src/components/ListItem.css

```diff
@@ -1,14 +1,29 @@
-.ListItem {
-	align-items: baseline;
+/* List item styling */
```

Comment:
Curious - totally fine to go with this approach, but why not utilize Bootstrap class names?

Reply:
Good Q! And honestly not sure what is best practice on formatting this so thank you for the heads up. I had bootstrap styling class names for half of my styling initially, but I realized I couldn't customize them with the class's not being easily identifiable in the CSS file. So I ended up consolidating them into the CSS file under regular CSS styling. But looking at the other's code I realize now I could have used a unique id classifier and still use the boot strap class names I'll definitely be looking into this and making update before demo!